eng/Versions.props

```diff
@@ -110,9 +110,6 @@
     <SystemRuntimeTimeZoneDataVersion>5.0.0-beta.20505.2</SystemRuntimeTimeZoneDataVersion>
     <SystemSecurityCryptographyX509CertificatesTestDataVersion>5.0.0-beta.20505.2</SystemSecurityCryptographyX509CertificatesTestDataVersion>
     <SystemWindowsExtensionsTestDataVersion>5.0.0-beta.20505.2</SystemWindowsExtensionsTestDataVersion>
-    <!-- Standard dependencies -->
-    <NETStandardLibraryVersion>2.2.0-prerelease.19564.1</NETStandardLibraryVersion>
```

Comment:
@terrajobst should we archive dotnet/standard repository since it's main branch isn't being consumed anywhere?